src/dotnet/commands/dotnet-new/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.DotNet.Tools.New
+{
+    internal static class LocalizableStrings
+    {
+        public static const string AppName = "dotnet new";
+
```

Comment:
Also, I'm sure you've seen the build errors, but it should be just `const` not `static const` ;)